ex1/ex1.c

```diff
@@ -10,8 +10,9 @@ int main(void)
 {
     int x = 100;
     //if this printf is below fork then it prints/runs before parent and child
-    printf("The initial value of x is %d.\n", x); 
+    printf("Initial value of x before fork() is %d.\n", x); 
```

Comment:
Good of you to make the point that this is the value of x _before the fork_, x proceeds normally until that fork happens.